Makefile

```diff
@@ -59,8 +69,17 @@ image-cephcsi: cephcsi
 	$(CONTAINER_CMD) build -t $(CSI_IMAGE_NAME):$(CSI_IMAGE_VERSION) deploy/cephcsi/image
 
 push-image-cephcsi: image-cephcsi
-	$(CONTAINER_CMD) push $(CSI_IMAGE_NAME):$(CSI_IMAGE_VERSION)
-
+	$(CONTAINER_CMD) tag $(CSI_IMAGE_NAME):$(CSI_IMAGE_VERSION) $(CSI_IMAGE_NAME)-$(ARCH):$(CSI_IMAGE_VERSION)
+	$(CONTAINER_CMD) push $(CSI_IMAGE_NAME)-$(ARCH):$(CSI_IMAGE_VERSION)
+
+# "docker manifest" requires experimental feature enabled
+push-manifest-cephcsi: export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
```
Experimental features provide early access to future product functionality. These features are intended only for testing and feedback as they may change between releases without warning or can be removed entirely from a future release.

Experimental features must not be used in production environments.
```
I would rather go with approach mentioned  in https://github.com/ceph/ceph-csi/issues/671#issuecomment-539912465 which  is currently used by rook

Reply:
https://github.com/estesp/manifest-tool says
`
UPDATE (Feb 2018): The Docker client PR #138 is merged, providing a docker manifest command that can replace the use of manifest-tool in most scenarios. Other follow-on PRs are in process to add functionality to docker manifest to make it completely functional for all multi-platform image use cases.
`
@travisn we can enable the experimental feature and use docker cli, any comments?

Reply:
My first attempt was using manifest-tool. It introduced extra efforts like downloading a third party tool and configuring executable path. Using docker cli directly is more concise.
But I'm not sure if this "docker manifest" command, as an experimental feature, may change in future.

Reply:
@Madhu-1 @travisn, shall we update this patch and turn to manifest-tool or keep using docker cli experimental feature?